load_default_pseudomonas_data.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Script to load default Pseudomonas data (in the files included in this
+# repository) into the adage-server database via management commands.
+
+# Import Pseudomonas annotation data
+python manage.py import_data ../data/PseudomonasAnnotation.tsv
```

Comment:
Using relative paths here means that this script must be run from a particular working directory. Can you document that in the comments at the head of the file? I guess an alternative would be to replace all of the relative paths with full paths -- perhaps one coded into an explicit constant.

If we are going in the direction of eliminating `fabric` in this way, we also need to make an issue to remind us to remove the configuration settings from `config.py.template` that hold all of these file names. That would need to be an issue because we'll still need those references to work until the transition to Docker deployment is complete.

Reply:
Right - I will make an issue for that but also document the fact that these are relative paths at the beginning of this file, since I think that is the most pragmatic approach.